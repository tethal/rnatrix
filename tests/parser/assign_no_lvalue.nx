fun main() {
    4 + 5 = 8;
}
