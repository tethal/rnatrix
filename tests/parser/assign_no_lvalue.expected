<string>:2:5: error: expected lvalue on the left side of assignment
    4 + 5 = 8;
    ^^^^^